packages/adminrouter/extra/src/includes/server/open/master.conf

```diff
@@ -109,6 +113,10 @@ location /acs/api/v1/auth/ {
 # Group: Metadata
 # Description: DC/OS GUI configuration (unauthenticated)
 location /dcos-metadata/ui-config.json {
+
+    set $upstream_tag Bouncer;
```

Comment:
This means that the name "Bouncer" will be exposed to users who use Grafana and the monitoring package, right? Perhaps "IAM" or similar is more appropriate, as Bouncer, I think is meant to be an internal name (though it does appear in our docs).

Reply:
Renamed to IAM.